ipatests/test_integration/test_replica_promotion.py

```diff
@@ -953,6 +953,8 @@ def test_hidden_replica_promote(self):
         tasks.wait_for_replication(
             self.replicas[0].ldap_connect()
         )
+        for srv in (self.master, self.replicas[0]):
+            tasks.dns_update_system_records(srv)
```

Comment:
This task ultimately calls `ipa dnsrecord-*` command, it should be enough to execute it on a single node.

Reply:
You're right. I switched to using one master and waiting for replication.